### ISSUE TITLE
Implement mailing list signup

### ISSUE THREAD

**Issue by rafeeJ** [Write-For-Change/emaildrafter#36]
I want to implement a modal pop up with a Mail Chimp(?) mailing list after the user has sent at least one email. 

The mailing list will be used to notify those who have signed up to the mailing list of new templates when they are added. 

Solution:
- [ ] Bootstrap modal popup.
- [ ] Use Mail Chimp API for sending automated emails when a new template is added.

**Comment by rafeeJ** [comment]
Created MailChimp account. Will explore API.

**Comment by benlellouch** [comment]
Created the modal popups which gives the user the option to subscribe to our newsletter (through mailchimp API).

**Comment by rafeeJ** [comment]
@benlellouch Awesome! Do you want to make a pull request onto develop branch so we can take a look?

**Comment by benlellouch** [comment]
@rafeeJ Yeah, of course! I need to add some vars to the env, how would I do that?

**Comment by rafeeJ** [comment]
@behradkoohy can add them as config vars on our development heroku box. You can send them to him :)

**Comment by benlellouch** [comment]
@rafeeJ Cool, I'll go annoy him in his room when he is free haha

**Comment by behradkoohy** [comment]
@rafeeJ @benlellouch :) added them into it
